cms/management/commands/subcommands/base.py

```diff
@@ -52,7 +52,7 @@ class SubcommandsCommand(BaseCommand):
     subcommand_dest = 'subcmd'
 
     def create_parser(self, prog_name, subcommand):
-        kwargs = {'cmd': self}
```

Comment:
@yakky I'm not sure if this is the right approach here, this is related to your comment on https://github.com/divio/django-cms/pull/6886#issuecomment-678237407

Reply:
removing it looks fine and has no ill effect.

As it's been removed in 2.1, we can safely drop it. Other packages (django-constance, for example) did the same